pseudo.css

```diff
@@ -0,0 +1,13 @@
+.block {
+	display: inline;
+}
+
+.inline {
+	display: block;
+	border: 1px solid slategray;
+	margin: 25px 25px 25px 25px;
```

Comment:
I revised this. I knew this already. I had written it this way at first, but the screen wrote in a weird way, so I wrote it out explicitly. Never went back and changed it back after reworking other things : /